core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -4106,12 +4190,24 @@ paths:
             'application/json':
               schema:
                 $ref: '#/components/schemas/WorkspaceMigrationMetadata'
+        400:
+          description: Some or all workspaces in billing project were ineligible for migration.
```

Comment:
This is just a single workspace, no?

Reply:
It is, this was a copy-paste error as well